src/config/default.yml

```diff
@@ -79,24 +79,22 @@ try:
       - taskId: '{{#as_slugid}}decision-task{{/as_slugid}}'
         task:
           created: '{{now}}'
-          deadline: '{{#from_now }}5 minutes{{/from_now}}'
+          deadline: '{{#from_now }}30 minutes{{/from_now}}'
           metadata:
             source: '{{{source}}}'
             owner: '{{owner}}'
-            name: Error message...
-            description: |
-              Error creating decision task...
+            name: Error Creating Decision Task
+            description: Error creating decision task
 
-          workerType: gecko-decision
+          workerType: "gecko-{{level}}-decision"
```

Comment:
This will probably fail for nss or vct jobs, as we don't have `gecko-nss-decision` workers, for example. It looks like `hg-worker` might be the mozilla-taskcluster equivalent of `github-worker`. I think that might be the appropriate default worker here?

Reply:
I'm not sure we have scopes for that.  Maybe `gecko-1-decision` is the best choice?

Reply:
Hmm, no, nss doesn't have that.  Hmm.  Let's leave it at gecko-decision, since that kinda worked.